packages/cli/src/commands/deploy.ts

```diff
@@ -5,6 +5,30 @@ import { BoosterConfig, Logger } from '@boostercloud/framework-types'
 import { Script } from '../common/script'
 import Brand from '../common/brand'
 import { logger } from '../services/logger'
+import { exec } from 'child-process-promise'
+import { wrapExecError } from '../common/errors'
+import * as path from 'path'
+
+
+let skipRestoreDependencies = false;
+
+async function pruneDependencies(config: BoosterConfig): Promise<void> {
+  try {
+    await exec('npm install --production', { cwd: projectDir(config) })
+  } catch (e) {
+    throw wrapExecError(e, 'Could not prune dev dependencies')
+  }
+}
+
+async function reinstallDependencies(config: BoosterConfig): Promise<void> {
```

Comment:
You can pass the parameter here like
```suggestion
async function reinstallDependencies(skipRestoreDependencies: boolean, config: BoosterConfig): Promise<void> {
```

And then, in the command part... (keep reading)